server/src/modules/mentorship-response/mentorship-response.model.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  Model,
+  Column,
+  Table,
+  DataType,
+  ForeignKey,
+  PrimaryKey,
+  AutoIncrement,
+} from 'sequelize-typescript';
+import { MentorshipRequest } from '../mentorship-request/mentorship-request.model';
+
+@Table({ tableName: 'mentorship_response' })
+export class MentorshipResponse extends Model<MentorshipResponse> {
+  private static ACCEPTED = 'Accepted';
+  private static DECLINED = 'Declined';
+
+  private static MENTORSHIP_RESPONSE_STATUS = DataType.ENUM(
+    MentorshipResponse.ACCEPTED,
+    MentorshipResponse.DECLINED,
```

Comment:
I think `mentorshipResponse` won't have a pending status, because will be created whenever the mentor accepts or rejects. `mentoshipRequest` should definitely have a pending status. Hmm maybe I am missing something.

Reply:
Ohhh sorry my bad, I thought this was on `MentorshipRequest`. You're right!